apps/judicial-system/web/src/routes/Admin/UserForm/UserForm.tsx

```diff
@@ -248,8 +252,8 @@ export const UserForm: FC<Props> = ({
               </Box>
             </Box>
             <Box className={styles.roleRow}>
-              {user.institution.id ===
-              '8f9e2f6d-6a00-4a5e-b39b-95fd110d762e' ? (
+              {user.institution.type ===
```

Comment:
Can we use `isPublicProsecutionOfficeUser` here?

Reply:
No, we can't. This form allows modifying the user role, so we cannot base this on the role the user happens to have.